.github/renovate.json

```diff
@@ -27,7 +27,7 @@
     },
     {
       "packageNames": ["com.google.apis:google-api-services-sqladmin"],
-      "allowedVersions": "/v1beta4-.*/"
+      "allowedVersions": "/v1-.*/"
```

Comment:
Let's leave v1beta4 in place. We do this for historical reasons. In short, v1beta4 should always be compatible with v1. To ensure that's the case, we continue using v1beta4 here.

Reply:
I switched back to v1beta4